packages/vue/src/components/organisms/SfGroupedProduct/SfGroupedProduct.stories.js

```diff
@@ -83,7 +83,7 @@ storiesOf("Organisms|GroupedProduct", module)
             priceSpecial: "",
             priceRegular: "$10,99",
             stock: 99,
-            qty: "0",
+            qty: 0,
```

Comment:
```suggestion
            qty: 1,
```

The minimum amount of the SfQuantitySelector inside _SfGroupedProductItem.html_ is `1`, so `0` doesn't make sense here.

But that made me thinking: Maybe there is a use-case where the user wants to allow a quantity of zero? In some shops you can remove a product by entering "0" in a quantity selector...

So for now I think you should just change all `qty: 0` to `qty: 1`. but maybe we need to discuss sometime whether we want to allow zeros in SfGroupdProductItem.

Reply:
@pspaczek I see you added props for `minQty` 👍 
Maybe also add them to the SfGroupedProduct stories, next to `stock`?

Reply:
For now, we can use 1. I set default 1 and add props minQty for SfGrouperProductItem and SfCollectedProduct. I think we shouldn't think what we should do when qty is 0, we should just allow set 0 :). Now you can do that by `qty-min` props :)